cloudpathlib/cloudpath.py

```diff
@@ -123,6 +123,7 @@ def __init__(cls, name, bases, dic):
                 not attr.startswith("_")
                 and hasattr(Path, attr)
                 and hasattr(getattr(Path, attr), "__doc__")
+                and getattr(getattr(Path, attr), "__doc__") is not None
```

Comment:
Couldn't both of these lines (125 and 126) just be simplified to `and getattr(getattr(Path, attr), "__doc__")`? This would both check if it exists, and that it casts as a boolean to positive (would exclude the `None` case). Not a big deal.

Reply:
Yep, good call. That should work as long as we add a [default to `getattr`](https://docs.python.org/3/library/functions.html#getattr) so it doesn't raise.